skins/oasis/modules/templates/LatestActivity_Index.php

```diff
@@ -1,28 +1,33 @@
 <section class="rail-module activity-module" id="wikia-recent-activity">
 
-	<h2 class="has-icon"><?= $activityIcon ?><?= $moduleHeader ?></h2>
+    <h2 class="has-icon"><?= $activityIcon ?><?= $moduleHeader ?></h2>
```

Comment:
Minor: The whole file should be using tabs per our PHP conventions.

Reply:
Fixed in 93c4c1a7be7